recipes/parro/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "parro",
+  "name": "Parro",
+  "version": "1.0.1",
```

Comment:
pls force this to 1.0.0 (any new recipe that we induct, we would like to start off with 1.0.0 version

Reply:
done